stack.yaml

```diff
@@ -63,15 +63,15 @@ extra-deps:
   - junit-xml-0.1.0.0
 
   - git: https://github.com/ttuegel/pipes-ghc-events.git
-    commit: 780b479300bd165f58b3bdd5f941ea81844fc937
+    commit: 4ab25e717658fa833aaf5efd76db38ccb1178f96
     subdirs:
       - eventlog2speedscope
-    # nix-sha256: 1qhpyx179n3zkaadvw8x411rqcvv12lvnnhkaxfa5fqm78bp2yzy
+    # nix-sha256: 05mpk1r04zsaiwzksbn0yd3s6fyah739rp6s4ir5nbb17b7j59pi
```

Comment:
@ttuegel I had some trouble with this PR, it took me a while to figure out I should change the hash here with the hash `nix/rematerialize.sh` complained about, and now it seems to be working. I'm wondering what I should have done in the first place, if there's an automatic way of getting these hashes or if I should look somewhere else.

Reply:
No, there's not an automatic way to do it. You did the right thing.